edx_proctoring/api.py

```diff
@@ -1058,6 +1062,11 @@ def _resolve_prerequisite_links(exam, prerequisites):
         'suggested_icon': 'fa-pencil-square-o',
         'in_completed_state': False
     },
+    ProctoredExamStudentAttemptStatus.waiting_for_app_shutdown: {
```

Comment:
ProctoredExamStudentAttemptStatus.waiting_for_app_shutdown is defined multiple times. Remove the other one